website/docs/components/input-tags/examples/basic.jsx

```diff
@@ -50,14 +56,20 @@ const Demo = () => {
             editable
             data-id={idx}
             onClick={handleEditTag}
+            onKeyDown={handleKey}
           >
             <InputTags.Tag.Text>{tag}</InputTags.Tag.Text>
             <InputTags.Tag.Close onClick={handleCloseTag} />
           </Tooltip.Trigger>
           <Tooltip.Popper>tag</Tooltip.Popper>
         </Tooltip>
       ))}
-      <InputTags.Value value={value} onChange={updateValue} onBlur={handleBlurInput} />
+      <InputTags.Value
+        value={value}
+        onChange={updateValue}
+        onBlur={handleBlurInput}
+        aria-label="input with tags"
```

Comment:
Could you synchronize with ux specialists and understand what is more important here, a placeholder or a label?

Reply:
Julia said that label is more important